Integrations/integration-PagerDuty.yml

```diff
@@ -0,0 +1,475 @@
+description: Alert and notify users using PagerDuty
+configuration:
+- display: API Key
+  name: APIKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Service Key (for triggering events only)
+  name: ServiceKey
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
```

Comment:
@ronykoz we are not using fetch here, you can take this off

Reply:
Done